### PR TITLE
Rework the Kafka Guide

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
Original [guide](https://cratedb.com/docs/guide/integrate/etl/kafka-connect.html) wasn't working anymore. It's similar, just dockerized and updated.

## Preview
https://cratedb-guide--124.org.readthedocs.build/integrate/etl/kafka-connect.html
## Checklist

 - [x] Link to issue this PR refers to (if applicable): https://github.com/crate/tech-writing/issues/428
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/integrate/etl/kafka-connect.rst [hunk 1]
```diff
@@ -1,504 +0,0 @@
...
-       AVRO_PRODUCER.produce(topic="metrics", value=create_metric())
-       AVRO_PRODUCER.flush()
-       time.sleep(0.25)
-
-This script can be run by installing the following dependencies and running it:
-
-.. code-block:: console
-
-   $ pip install "confluent-kafka[avro]" "avro-python3"
-   $ python simulator.py
```
Comment (amotl) [inline docs/integrate/etl/kafka-connect.rst:222]: I am sad to see this Python snippet removed. Maybe we can bring it back in one way or another, if not within this tutorial, maybe at another spot.

File: docs/integrate/etl/kafka-connect.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
+(kafka-connect)=
+# Data Ingestion Pipeline with Kafka and CrateDB
+
+This guide describes a dockerized procedure for integrating CrateDB with Kafka
+Connect. By following these steps, you will set up a pipeline to ingest data 
+from Kafka topics into CrateDB seamlessly.
```
Comment (amotl) [inline docs/integrate/etl/kafka-connect.md:6]: Active voice, very good.

File: docs/integrate/etl/kafka-connect.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+This results in the following architecture:
+
+
+![image](/_assets/img/integrations/kafka-connect.png){width=650px}
+
+## Setup
+To illustrate how this architecture can be used, we will create a scenario 
+where we have machine sensor data from a series of weather stations being 
+ingested into a Kafka topic. This data could be used in a reactive sense: for
```
Comment (amotl) [inline docs/integrate/etl/kafka-connect.md:53]: Active voice?
```suggestion
To illustrate how this architecture can be used, let's create a scenario 
where machine sensor data from a series of weather stations is 
ingested into a Kafka topic. This data could be used in a reactive sense: for 
```

File: docs/integrate/etl/kafka-connect.md [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+
+## Setup
+To illustrate how this architecture can be used, we will create a scenario 
+where we have machine sensor data from a series of weather stations being 
+ingested into a Kafka topic. This data could be used in a reactive sense: for 
+example, a micro-controller could consume from this topic to turn on air 
+conditioning if the temperature were to rise above a certain threshold. Aside
+this use of the data, we want to ingest it into CrateDB. This allows 
+us to do long term data analytics, like predicting the weather trends for 
+example. Payload from each sensor may look like this:
```
Comment (amotl) [inline docs/integrate/etl/kafka-connect.md:58]:
```suggestion
conditioning if the temperature were to rise above a certain threshold. Aside
this use of the data, it should be stored into CrateDB. This allows 
to do long term data analytics, like predicting the weather trends for 
example. Payload from each sensor may look like this:
```

**Review by amotl — APPROVED** [review]
Hi. Thanks a stack for bringing this up to speed. I did not verify the procedure, but the document looks good. On two spots, I added suggestions about converting to active voice, but in general it is good to go.

File: docs/integrate/etl/kafka-connect.rst (hunk 1 above)
Comment (matkuliak) [reply docs/integrate/etl/kafka-connect.rst:222]: Absolutely. The current setup is pretty crude, I'm sure we can improve it with a similar script

**Pull request merged**